app/scraper/loot/google_games.py

```diff
@@ -38,7 +38,12 @@ def get_duration() -> OfferDuration:
 
     @staticmethod
     def scrape(driver: WebDriver) -> list[Offer]:
-        return GoogleGamesScraper.read_offers_from_page(driver)
+        offers = GoogleGamesScraper.read_offers_from_page(driver)
+        categorized_offers = GoogleGamesScraper.categorize_offers(offers)
+        filtered = list(
+            filter(lambda offer: offer.category != Category.DEMO, categorized_offers)

```

Comment:
Can this be done in a more generic way (not for every scraper seperately)?

Reply:
Could be done, just use this as the default implementation in scraper.py.
I wanted this PR to have no functional changes.

In general I'd suggest doing the filtering when reading from the DB, not when writing to it (at the moment demos are not written to the DB).

Reply:
Okay, let's leave it for a future PR.